src/components/molecules/Visualizer/hooks.ts

```diff
@@ -249,6 +259,97 @@ export default ({
     pageview(window.location.pathname);
   }, [isPublished, enableGA, trackingId]);
 
+  const [overriddenAlignSystem, setOverrideAlignSystem] = useState<WidgetAlignSystem | undefined>(
```

Comment:
`Visualizer/hooks.ts` is now too large. Please separate hooks associated with the alignment system into smaller hooks.

Reply:
Updated. Separated into `useWidgetAlignSystem.ts`.